tests/conftest.py

```diff
@@ -379,7 +379,10 @@ def factory(
         secret=None,
         custom_data=None,
         sc_name=None,
-        reclaim_policy=constants.RECLAIM_POLICY_DELETE
+        reclaim_policy=constants.RECLAIM_POLICY_DELETE,
+        replica=3,
```

Comment:
we can set replica to None as default so pool creation will use the default value in case replica is not provided

Reply:
By default, replica is 3 in cephblockpool.yaml as well. So I thought to keep as 3 only